### ISSUE TITLE
Spigot version is not the new build.

### ISSUE THREAD

**Issue by EllriNidhogg** [PenToPaper/TPPets#5]
I was going to pull and fix and compile, but it appears you have already fixed the cats issue. Spigot doesn't have the latest build.

**Comment by PenToPaper** [comment]
Yes, I will be doing some better testing tomorrow and will be pushing the new version to spigot then.
